### ISSUE TITLE
Cannot understand frame header exception when receiving messages from Javascript

### ISSUE THREAD

**Issue by ahychu84** [vtortola/WebSocketListener#139]
This exception is getting thrown from WebSocketConnectionRfc6455 when trying to parse the header of a message that is being sent from a wss connection from javascript. The code associated is super simple and all I'm sending is an string which is in the following format: "AUTHENTICATE:{token}". I have had this app running with a very old version of this websocketlistener and this only happened after upgrading. I have tested through IE, Edge and Chrome and all yield the same exception. Any help would be appreciated.

Upon further research, this issue seems to only be an issue when using wss to connect and sending a message after the connection is made. If wss works if I'm just receiving data from the server

**Comment by vtortola** [comment]
Hi,

That's odd. Have you tried to create a small prof of concept and narrow down the problem?

**Comment by ahychu84** [comment]
yep I found it weird too. I tried running this in dotnet and mono and they both yield the same error. From what I'm seeing the error gets thrown in the WebSocketListener class. I'm gonna try using another websocket lib to test to see if I receive the same error

I've determined this bug was introduced after version 2.2.2

**Comment by vtortola** [comment]
I will try to put together a small repro project.

**Comment by ahychu84** [comment]
Our in house environment is all linux (CentOS 7) so I have tested this using mono and dotnet

**Comment by ahychu84** [comment]
@vtortola any luck ?
